hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java

```diff
@@ -497,12 +505,14 @@ public HapiSpec creationHappyPath() {
                                 .hasSupplyType(TokenSupplyType.FINITE)
                                 .hasEntityMemo(memo)
                                 .hasName(saltedName)
+                                .hasMetadata(metaData)
                                 .hasTreasury(TOKEN_TREASURY)
                                 .hasAutoRenewPeriod(THREE_MONTHS_IN_SECONDS)
                                 .hasValidExpiry()
                                 .hasDecimals(1)
                                 .hasAdminKey(PRIMARY)
                                 .hasFreezeKey(PRIMARY)
+                                .hasMetadataKey(METADATA_KEY)
```

Comment:
This will not work until we make changes to `TokenGetInfoHandler`. Did you run to see if it works ?

Reply:
You're right. Will roll back until we implement changes to `TokenGetInfoHandler` 👍